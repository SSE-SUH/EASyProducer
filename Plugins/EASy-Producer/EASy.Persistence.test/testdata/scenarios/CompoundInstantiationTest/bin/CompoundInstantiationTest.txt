notNestedVar = Not nested
nestedVar = Nested, layer 1
nestedVar = 1. nested, layer 2
nestedVar = 2. nested, layer 2
