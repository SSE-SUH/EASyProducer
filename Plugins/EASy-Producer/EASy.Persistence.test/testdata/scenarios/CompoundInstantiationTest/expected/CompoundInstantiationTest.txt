notNestedVar = Not nested
cmp1.nestedVar = Nested, layer 1
cmp2.slot1.nestedVar = 1. nested, layer 2
cmp2.slot2.nestedVar = 2. nested, layer 2
